hazelcast/src/main/java/com/hazelcast/ringbuffer/Ringbuffer.java

```diff
@@ -67,41 +77,44 @@
     long capacity();
 
     /**
-     * Returns number of items in the ringbuffer.
-     *
-     * If no ttl is set, the size will always be equal to capacity after the head completed the first loop
-     * around the ring. This is because no items are getting retired.
+     * Returns number of items in the Ringbuffer.
+     * <p>
+     * If no ttl is set, the size will always be equal to capacity after the head completed the first loop around the ring.
```

Comment:
Keep it for now, but prefer javadoc up to (around) 72 characters per line in the future. It possibly makes it more readable. I know a lot of the current javadoc is not adhering to this rule.